src/Text/Pandoc/Readers/Markdown.hs

```diff
@@ -1735,6 +1736,17 @@ source = do
 linkTitle :: PandocMonad m => MarkdownParser m Text
 linkTitle = quotedTitle '"' <|> quotedTitle '\''
 
+-- Github wiki style link, with optional title
+githubWikiLink :: PandocMonad m => MarkdownParser m (F Inlines)
+githubWikiLink = try $ do
+  guardEnabled Ext_markdown_github_wikilink
+  string "[["
+  skipSpaces
+  let allUntil = manyTillChar (satisfy $ const True)
```

Comment:
`satisfy $ const True` is just `anyChar`, right?

Reply:
Also: do we really want anyChar here?  Suppose you have `[[` followed by sixteen paragraphs of text, each separated by a blank line.

Reply:
In pandoc's links, the link description can contain inline formatting.  It would be natural to extend that feature here, though I don't know how GitHub's parsing actually works.

Reply:
I just tried in https://github.com/jgm/pandoc/wiki/Sandbox/_edit so yeah, bold and italics appear supported... but not links of course.